### PR TITLE
ncm-metaconfig: add lvm_conf config support

### LINKED ISSUE #1566: ncm-metaconfig: add support to manage lvm.conf
Add support to manage  /etc/lvm/lvm.conf possibly using /etc/lvm/lvmlocal.conf

see man lvm.conf.

### PR DESCRIPTION
**Pull request by aka7**
ability to set lvm configuration. Using lvmlocal.conf.

All default options supported taken from output lvmconfig --typeconfig default on rhel7 and 8.

Fixes #1566

### CODE REVIEW

**Comment by aka7** [comment]
I can't seem to get the tests to pass, if someone can help me with this I would appreciate it. @jrha

File: ncm-metaconfig/src/main/metaconfig/lvm_conf/tests/regexps/config [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+^\}$
+^local \{$
+^\}$
+^dmeventd \{$
+^\}$
+^activation \{$
+^    activation_mode=degraded$
+^\}$
+^global \{$
+^    activation=0$
```
Comment (jrha) [inline ncm-metaconfig/src/main/metaconfig/lvm_conf/tests/regexps/config:16]: I changed this to `activation` to match above

File: ncm-metaconfig/src/main/metaconfig/lvm_conf/tests/profiles/config.pan [hunk 2]
```diff
@@ -0,0 +1,8 @@
+object template config;
+
+include 'metaconfig/lvm_conf/config';
+
+prefix "/software/components/metaconfig/services/{/etc/lvm/lvmlocal.conf}/contents";
+
+"global" = dict('activation', 0);
```
Comment (jrha) [inline ncm-metaconfig/src/main/metaconfig/lvm_conf/tests/profiles/config.pan:7]: I changed this from `activation_mode` to `activation` to get the tests to pass, but did you actually want `event_activation` here?

**Review by jrha — APPROVED** [review]
Okay I'm now happy with this.
@aka7 can you check that the changes I've made make sense to you.

**Pull request merged**